### PR TITLE
Update icfree-ml to 2.3.3

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`icfree-ml`](https://bioconda.github.io/recipes/icfree-ml/README.html): **2.3.2** &rarr; **2.3.3**

[install with bioconda](http://bioconda.github.io/recipes/icfree-ml/README.html) [Conda](https://anaconda.org/bioconda/icfree-ml/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/icfree-ml`](https://github.com//bioconda/bioconda-recipes/tree/bump/icfree_ml/recipes/icfree-ml) (click to view/edit other files)
Summary | Design of experiments (DoE) and machine learning packages for the iCFree project
Home | [https://github.com/brsynth/icfree-ml](https://github.com/brsynth/icfree-ml)
Releases |[https://github.com/brsynth/icfree-ml/tags](https://github.com/brsynth/icfree-ml/tags)
Recipe Maintainer(s) | @tduigou
Author | `@brsynth`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/icfree-ml/meta.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ requirements:
     - biopython
     - pysbol2
     - blast
+    - pandas
```
Comment (martin-g) [inline recipes/icfree-ml/meta.yaml:28]:
```suggestion
    - pandas
    - pydoe2
```

**Comment by tduigou** [comment]
Hi @martin-g, I try to append an additional change to fix the recipe: https://github.com/bioconda/bioconda-recipes/pull/49138

Any help is welcome
Thanks

**Comment by tduigou** [comment]
Hi @martin-g, it looks like the merged code from [2e676f5](https://github.com/bioconda/bioconda-recipes/pull/49037/commits/2e676f5401a27530a77c00312459972162e80a97) fix the issue, but then the CI/CD fails after bumping the build number.

**Comment by martin-g** [comment]
All should be OK now with https://github.com/bioconda/bioconda-recipes/pull/49147

**Pull request closed without merging**